frontend/src/components/home/midsection/lower_mid.jsx

```diff
@@ -1,62 +1,84 @@
 import React, { Component } from "react";
 import "./mid.css";
 import { Container, Row, Col } from "react-bootstrap";
+import { connect } from "react-redux";
+import { events } from "../../../actions";
+import { findNextUpcomingEvent } from "../../events/utils";
+import { getSorted, getUpcomingEvents } from "../../../reducers/selectors";
+import Moment from "react-moment";
 import footerImage1 from "../../../assets/footerImage1.jpg";
 import footerImage2 from "../../../assets/Group_at_a_table_YMIM.png";
 import footerImage3 from "../../../assets/Group_pic_YMIM.png";
 import footerImage4 from "../../../assets/chicago-illinois-skyline-skyscrapers-161963.jpeg";
 
 class LowerMid extends Component {
+  componentDidMount() {
+    this.props.fetchEvents();
+  }
+
   render() {
+    const defaultContent =
+      "Check back again soon for what's is coming up next for YMIM. If you have an event that you think YMIM should be a part of please please email: Founder@theymim.org or call: 773.941.1200";
+    const upcomingEvents = this.props.upcomingEvents;
+    let nextEvent;
+    if (upcomingEvents.length) {
+      nextEvent = findNextUpcomingEvent(upcomingEvents);
+    }
+    const lowermidSection = (
+      <>
+        <h2 className="text-center heading">What's New?</h2>
+        {nextEvent ? (
+          <h2 className="text-center sub-heading">
+            <Moment format="llll">{nextEvent.start.local}</Moment>
+            {" - " + nextEvent.name.text}
+          </h2>
+        ) : (
+          ""
+        )}
```

Comment:
Style choice, I'd recommend avoiding the ternary operator when one of the true/false clauses are empty. Instead use `&&`.

e.g.: `{ nextEvent && (<h2...../>) }`